### DISCUSSION TITLE
Webhook Support on Macros

### DISCUSSION THREAD

**Discussion by luizeof** [chatwoot/chatwoot#9727] [Feature Requests]
The possibility of an agent pressing a button (macro) and us being able to access external automations with Zapier, N8N, etc. is incredible.

There is already an issue that perfectly addresses this feature. Could you reopen it for us to discuss? https://github.com/chatwoot/chatwoot/issues/5968

**Comment by pranavrajs** [comment]
We will continue the discussion over https://github.com/chatwoot/chatwoot/issues/5968
